app/javascript/app/components/ndcs/ndcs-explore-table/ndcs-explore-table-selectors.js

```diff
@@ -42,6 +49,50 @@ export const getDefaultColumns = createSelector(
   }
 );
 
+export const tableGetSelectedData = createSelector(
+  [getIndicatorsParsed, getCountries],
+  (indicators, countries) => {
+    if (!indicators || !indicators.length || !indicators[0].locations) {
```

Comment:
Please add `countries` guard to this selector:
```suggestion
    if (!indicators || !indicators.length || !indicators[0].locations || !countries) {
```